instructure/canvas-android
# [Flutter-Parent][MBL-13956] Encrypted prefs

Everything under the `plugin` directory is copied from the SharedPreferences flutter plugin. There was a small tweak to use EncryptedSharedPreferences instead, but 99% of the code is the same as the initial plugin.

To test:
Run the app and login (or just run the app if you had previous logins).
Open an instance of android studio, and navigate to the device file explorer
Navigate to `data/data/com.instructure.parentapp/shared_prefs/`
* Verify there is a FlutterSharedPreferences.xml that doesn't contain any user/login data
* Verify there is a FlutterEncrypedSharedPreferences.xml that contains user data that is not readable (it's encrypted)
